tasks.py

```diff
@@ -383,9 +383,12 @@ def test_addons():
 
 
 @task
-def test(all=False):
+def test(all=False, syntax=False):
```

Comment:
None aside from consistency with the previous usage. This PR has already been merged though.